tests/integration/api_packages_container_test.go

```diff
@@ -84,15 +84,17 @@ func TestPackageContainer(t *testing.T) {
 			Token string `json:"token"`
 		}
 
-		authenticate := []string{`Bearer realm="` + setting.AppURL + `v2/token",service="container_registry",scope="*"`}
+		wwwAuthenticateValues := func() []string {
+			return []string{`Bearer realm="` + setting.AppURL + `v2/token",service="container_registry",scope="*"`}
+		}
```

Comment:
Why a func?

Reply:
Because the AppURL might change during the tests. So it should always read the current AppURL

Reply:
Read the code again, now we don't need this function. So just rename it to `defaultAuthenticateValues`

Reply:
Yep, approved already 👍